rero_ils/modules/notifications/templates/email/at_desk/fre.txt

```diff
@@ -0,0 +1,17 @@
+Exemplaire au bureau de prêt
+
+{% for loan in loans -%}
+Date: {{ loan.creation_date }}
+Titre: {{ loan.document.title_text }}
+Code-barre: {{ loan.document.barcode }}
```

Comment:
Code-barre -> Code-barres

Reply:
@pronguen @PascalRepond 
@jma remark seems correct. Is it ok to change in any notification french templates ? 
What's about other languages ?

Reply:
Of course!